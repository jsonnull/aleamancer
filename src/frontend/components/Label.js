// @flow
import styled from 'styled-components'
import { fontSize } from 'frontend/styles/common'

const Label = styled.label`
  font-size: ${fontSize.small};
  color: ${props => props.theme.colorSecondary};
  font-weight: ${props => (props.theme.name == 'dark' ? 'normal' : 'bold')};
  display: block;
`

export default Label
