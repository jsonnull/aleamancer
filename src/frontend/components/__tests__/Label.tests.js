// @flow
import React from 'react'
import renderer from 'react-test-renderer'
import Label from 'frontend/components/Label.js'

describe('Label component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Label />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
