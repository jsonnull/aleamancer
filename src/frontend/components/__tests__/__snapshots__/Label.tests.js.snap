// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component renders correctly 1`] = `
.c0 {
  font-size: 1.3rem;
  font-weight: bold;
  display: block;
}

<label
  className="c0"
/>
`;
